python/cudf/cudf/tests/test_groupby.py

```diff
@@ -666,8 +675,10 @@ def test_groupby_multi_agg_multi_groupby():
     gdg = gdf.groupby(["a", "b"]).agg(["sum", "max"])
     assert_eq(pdg, gdg)
 
-
+@pytest.mark.xfail()
```

Comment:
We can't just xfail this. We need to fix the underlying problem.

Reply:
Agreed. The underlying problem is new `quantile` logic. New pandas just doesnt' line up for us anymore. Their new logic is clean and ours is sparse. It should be an easy fix and a good thing for us. I have removed all of the other attempts at solving the issue in the `groupby` tests as I eliminated the possibility that it was a column order issue or datetime/timezone issues. After I've complete the `DataFrame.__repr__()` refactor, I'll button up `quantile`